src/sample_utils.py

```diff
@@ -8,10 +8,19 @@
 import json
 import time
 import resource_uri_utils
-from azure.common.credentials import ServicePrincipalCredentials
-from azure.mgmt.netapp import AzureNetAppFilesManagementClient
+from azure.core.exceptions import HttpResponseError, \
+    ResourceNotFoundError
+from azure.identity import ClientSecretCredential
+from azure.mgmt.netapp import NetAppManagementClient
+from azure.mgmt.netapp.models import ReplicationStatus
 from datetime import datetime
-from msrestazure.azure_exceptions import CloudError
+from enum import Enum
+
+class MirrorState(Enum):
```

Comment:
>MirrorState [](http://example.com/codeflow?start=6&length=11)

Let's adopt the mirror_state naming convention which aligns more with Python instead of Pascal case.